src/main/java/ml/shifu/shifu/core/dtrain/gs/GridSearch.java

```diff
@@ -282,6 +282,15 @@ private Object convertItemValue(Map<String, MetaItem> metaWarehouse, String item
         itemValueStr = itemValueStr.trim();
         if(itemMeta.getType().equals("text")) {
             return itemValueStr;
+        } else if(itemMeta.getType().equals("integer")) {
```

Comment:
itemMeta.getType().equals("integer") || itemMeta.getType().equals("int") I think is better to make it be stronger

Reply:
@zhangpengshan will update it!